### ISSUE TITLE
Wait for bitcoind to be fully synced on startup

### ISSUE THREAD

**Issue by elsirion** [fedimint/fedimint#5860]
Otherwise we may sync way too many block headers, making the initial sync slower.

**Comment by Kodylow** [comment]
Yes please, big pain point for guardian setup. would also like to be able to reflect this in the UI

**Comment by elsirion** [comment]
TODO: test new setup process behavior.

**Comment by elsirion** [comment]
https://github.com/fedimint/fedimint/pull/7523 makes this one mostly redundant.
